presets/exhibition-preset/src/creators/image-url-slide.tsx

```diff
@@ -2,13 +2,19 @@ import { imageServiceCreator } from "@manifest-editor/creators";
 import { CreatorDefinition, CreatorFunctionContext } from "@manifest-editor/creator-api";
 import { ImageService, InternationalString } from "@iiif/presentation-3";
 
-export const imageUrlSlide: CreatorDefinition = {
+export const imageUrlSlideCreator: CreatorDefinition = {
   ...imageServiceCreator,
-  id: "@exhibitions/image-service-creator",
+  id: "@exhibitions/image-url-creator",
```

Comment:
Also the 2 Id's where the same! but changing that on its own didnt make them show 🤷🏻